actor-typed-tests/src/test/java/org/apache/pekko/actor/typed/eventstream/EventStreamDocTest.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.pekko.actor.typed.eventstream;
```

Comment:
can you add a standard Apache header here? not the one that mentions Akka

Reply:
Added